apps/src/componentLibrary/fontAwesomeV6Icon/FontAwesomeV6Icon.story.tsx

```diff
@@ -37,11 +44,22 @@ DefaultFontAwesomeV6Icon.args = {
   title: 'check',
 };
 
-export const ClassExampleFontAwesomeV6Icon = SingleTemplate.bind({});
-ClassExampleFontAwesomeV6Icon.args = {
-  iconStyle: 'light',
-  iconName: 'brands fa-microsoft',
-  title: 'microsoft',
+export const ClassNameBrandsIconExampleFontAwesomeV6Icon =
+  MultipleTemplate.bind({});
+ClassNameBrandsIconExampleFontAwesomeV6Icon.args = {
+  components: [
+    {
+      iconStyle: 'light',
+      iconName: 'microsoft',
+      className: 'fa-brands',
```

Comment:
Maybe out of scope for this change, but if we're taking care of the `fa-` for the icon name, should we also take care of it for the class name?

Reply:
Not necessary I think since this className can be any string, including not fontAwesome related custom classNames

Reply:
I see.  I think the only issue is that to understand how to use the brand ones, you basically need to dig into the implementation to understand when the prefix is added or not.

Reply:
Yeah, it is. Though I wonder how Brand Icons are shown in Figma, cause I can't find one. I think it should mark something like font family - Font Awesome 6 Brands which will help one to understand that it's a brand icon and it needs fa-brands className